docker-compose.test.yml

```diff
@@ -190,9 +192,9 @@ services:
         TENSORFLOW_PACKAGE: tf-nightly-gpu==1.14.1-dev20190408
         KERAS_PACKAGE: git+https://github.com/keras-team/keras.git
         PYTORCH_PACKAGE: torch-nightly
-        MXNET_PACKAGE: mxnet-cu100==1.4.0.post0
+        MXNET_PACKAGE: mxnet-cu100 --pre
```

Comment:
why not use the same 1.5.0b20190412 as above here?

Reply:
See above answer.